config/secrets.yml

```diff
@@ -57,6 +57,8 @@ default: &default
 
   show_pame_statistics: false
 
+  disallow_all_web_crawlers: <%= ENV["DISALLOW_ALL_WEB_CRAWLERS"] %>
```

Comment:
Actually I think if this is just going to be a true/false thing, we could just put it in the secrets as it's nothing sensitive, no need for an environment variable. What do you think?